projects/packages/forms/src/blocks/contact-form/block.json

```
@@ -3,7 +3,7 @@
 	"apiVersion": 3,
 	"name": "jetpack/contact-form",
 	"title": "Form",
-	"description": "Create forms to collect data from site visitors and manage their responses.",
+	"description": "Create forms with layout support to collect data from site visitors and manage their responses.",
```

Let's not complicate the description because of this, layouts are just a design detail.


```suggestion
	"description": "Create forms to collect data from site visitors and manage their responses.",
```